### ISSUE TITLE
Docker-compose does not work solr can not be run as root

### ISSUE THREAD

**Issue by Dantali0n** [projectblacklight/blacklight#2557]
The docker-compose scripts don't work and can't be used to get a working installation due to solr_wrapper refusing to start as root.

**Comment by dkinzer** [comment]
@Dantali0n using docker-compose up to build the app is not currently supported.  The solr image is merely used for testing purposes on the CI environment and locally because in the past solr_wrapper has been unreliable.

I did just now try to spin up the environment using docker-compose up and I can see there are a lot unresolved issues with it, including:
* ALPINE_RUBY_VERSION needs to be updated to be the same as what is in the Gemfile.
* the shared-mime-info library is missing from the installed image packages.
* solr_wrapper should not  even be spinning up at all since the build should rely on the solr container  that spins up as part of the docker-compose up process.

Currently the documented way of using the solr docker instance is by itself https://github.com/projectblacklight/blacklight/wiki/Testing-and-Developing-Blacklight

That said I think I may have a fix for the issues I've listed above and I'll post a PR to fix them.
